kth-largest-element-in-a-stream.py

```diff
@@ -1,23 +1,23 @@
-class KthLargest:
-
-    def __init__(self, k: int, nums: List[int]):
-        nums.sort()
-        self.k = k
-        track = nums[:]
```

Comment:
It is better if you use pythons .copy() method. It will make you code more readable